benchmarks/nnls/solvers/cd.py

```diff
@@ -0,0 +1,78 @@
+import numpy as np
+
+from numpy.linalg import norm
+
+from benchopt.base import BaseSolver
+from benchopt.util import safe_import
+
+
+with safe_import() as solver_import:
+    from scipy import sparse
+    from numba import njit
+
+
+if solver_import.failed_import:
+
+    def njit(f):  # noqa: F811
+        return f
+
+
+class Solver(BaseSolver):
+    name = "cd"
+
+    install_cmd = 'pip'
+    requirements = ['numba']
```

Comment:
You might need to add "scipy" here

Reply:
good catch. I added this also to the cd.py version in Lasso solvers, as this error was due to a copy pasted from that file.